lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -524,9 +524,13 @@ def _section_cohort_management(course, access):
 def _section_discussions_management(course, access):
     """ Provide data for the corresponding discussion management section """
     course_key = course.id
+    enrollment_track_schemes = available_division_schemes(course_key)
     section_data = {
         'section_key': 'discussions_management',
         'section_display_name': _('Discussions'),
+        'is_hidden': (not is_course_cohorted(course_key) and
```

Comment:
No never mind-- I thought it was a class name being applied to the section. Thanks for the explanation!